youth_data_book.R

```diff
@@ -0,0 +1,195 @@
+library(dplyr)
+library(tidyr)
+library(broom)
+library(feather)
+library(foreign)
+library(sas7bdat)
+
+##########
+# datasets
+##########
+# 1) "1987_2015_labour_force_survey"
+# 2) "2010_general_social_survey"
+# 3) "2010_imdb_tax_file"
+# 4) "2011_general_social_survey"
+# 5) "2012_general_social_survey"
+# 6) "2012_program_for_international_assessment_of_adult_comptencies"
+# 7)  "2013_general_social_survey"
+# 8) "2014_cananda_financial_capabilities_survey"
+# 9) "2014_employment_insurance_coverage_survey"
+# 10) "2014_general_social_survey"
+# 11) "2015_ontario_student_drug_use_and_health_survey"
+#12) "year_ontario_social_assistance_database"
+##########################################################################
+
+##########
+# function will loop through files and get the attributes of the columns names as well
+# as the summary for that column (and number of NAs)
+##########
+
+path_to_data = "data/youth_data_book"
```

Comment:
It doesnt exist yet because we haven't merged it yet right? I think in the original directions I asked you to download the folder youth_data_book and store the data in there.